internal/binutils/binutils.go

```diff
@@ -140,7 +144,87 @@ func initTools(b *binrep, config string) {
 		b.addr2line, b.addr2lineFound = findExe("gaddr2line", append(paths["addr2line"], defaultPath...))
 	}
 	b.nm, b.nmFound = findExe("nm", append(paths["nm"], defaultPath...))
-	b.objdump, b.objdumpFound = findExe("objdump", append(paths["objdump"], defaultPath...))
+	// MacOS binary is named objdump, Linux binary is named llvm-objdump-version
+	b.objdump, b.llvmObjdumpFound, b.objdumpFound = findObjdump(append(paths["llvm-objdump"], append(paths["objdump"], defaultPath...)...))
+}
+
+// findObjdump finds all binaries named objdump, llvm-objdump*. If multiple LLVM objdump are found,
+// it picks objdump with the latest major version. If no LLVM objdump is found, returns the path to
+// any binary named objdump, if found.
+func findObjdump(paths []string) (string, bool, bool) {
+	allObjdump := findAllObjdump("llvm-objdump*", paths)
+
+	if objdump, objdumpFound := findExe("llvm-objdump", paths); objdumpFound {
+		allObjdump = append(allObjdump, objdump)
+	}
+
+	if objdump, objdumpFound := findExe("llvm-objdump-9", paths); objdumpFound {
```

Comment:
Not very sure we want to support Debian-flavored `/usr/bin/llvm-objdump-9`

Users can run update-alternatives if they do want `/usr/bin/llvm-objdump`

Reply:
gLinux has llvm-ovjdump-7, 8 & 9 in path.

Looks like this might be gLinux specific. They are just symlinks to llvm-objdump is various version locations.

Reply:
I am not sure we want to manage the selection of the versioned binary names. I think it's reasonable to assume that there is unversioned symlink (llvm-objdump) available and if not then user should make sure that there is.

Reply:
I'll remove this check.